convert_zpl_format_to_png_format/src/main/java/com/testsigma/addons/web/ZplToPngConvertor.java

```diff
@@ -0,0 +1,109 @@
+package com.testsigma.addons.web;
+
+import com.testsigma.sdk.ApplicationType;
+import com.testsigma.sdk.Result;
+import com.testsigma.sdk.WebAction;
+import com.testsigma.sdk.annotation.Action;
+import com.testsigma.sdk.annotation.RunTimeData;
+import com.testsigma.sdk.annotation.TestData;
+import lombok.Data;
+import org.openqa.selenium.NoSuchElementException;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+@Data
+@Action(actionText = "Convert ZPL file-path(ex: \C:\Users\\sam.isaac\\Downloads\\zplpng.png) into PNG file with name (Ex:output.png) file-name and output file path output-file-path and store " +
```

Comment:
Whay can't the file uploaded from uploads here?
Change the Action text,
File Action: Convert ZPL file File-Path(ex: /Users/sam.isaac/Downloads/zplpng.png) into PNG file with name (Ex:output.png) File-Name and output file path output-file-path and store file path in a runtime variable Variable-Name

Reply:
Cx asked for local execution so i build it in this way.First they will convert base64 to zpl and then zpl to png